File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    drag = []
    lux = []
    luy = []
    for i in range(len(wallpaper)):
        for j in range(len(wallpaper[i])):
            if wallpaper[i][j] == '#':
                lux.append(i)
                luy.append(j)
                
    drag.append(min(lux))
    drag.append(min(luy))
    drag.append(max(lux)+1)
    drag.append(max(luy)+1)
        
    return drag

File: Java/프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.java
class Solution {
    public int[] solution(String[] wallpaper) {
        int lux, luy;
        int rdx, rdy;
        int row = wallpaper.length;
        int col = wallpaper[0].length();
        char[][] grid = new char[row][col];
        
        for (int i = 0; i < row; i++) {
            grid[i] = wallpaper[i].toCharArray();
        }
        
        lux = row;
        luy = col;
        rdx = 0;
        rdy = 0;
        
        for (int i = 0; i < row; i++) {
            for (int j = 0; j < col; j++) {
                if (grid[i][j] == '#') {
                    lux = Math.min(lux, i);
                    luy = Math.min(luy, j);
                    rdx = Math.max(rdx, i);
                    rdy = Math.max(rdy, j);
                }
            }
        }
        
        int[] answer = {lux, luy, rdx+1, rdy+1};
        return answer;
    }
}